recipes/recipes_emscripten/lz4-c/build.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+set -ex
+
+export CFLAGS="${CFLAGS} -O3 -fPIC"
+export LDFLAGS="${LDFLAGS} -Wl,-rpath,${PREFIX}/lib -L${PREFIX}/lib"
```

Comment:
We can probably remove this line. `-rpath` is not supported and the default `LDFLAGS` should work well.